analyzers/its/sources/Net6/Net6/MutexAclTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Net6
```

Comment:
this will probably fail the PR, I'll send an update after the 2nd review (it's just a formality)